lib/setup/database.rb

```diff
@@ -50,7 +50,7 @@
 
     create_table :stops, force: true do |t|
       t.boolean :active, default: true
-      t.integer :hastus_id
+      t.string  :hastus_id
```

Comment:
We don't have migrations in this app, so this "migration" has actually already been "run" in prod.